src/schema.rs

```diff
@@ -0,0 +1,65 @@
+table! {
+    laporan (id) {
+        id -> Uuid,
+        created_date -> Timestamptz,
+        updated_date -> Nullable<Timestamptz>,
+        judul -> Varchar,
+        nomor -> Nullable<Varchar>,
+        isi -> Text,
+        status -> Laporan_status,
```

Comment:
Suggestion, May need to declare the laporan_status enum type in Rust too

Reply:
The `naming` should also be reconsidered, this will be tomorrow's discussion, good point @allisandrana